Hymns/Display/HymnLyricsView.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftUI
+import Resolver
+
+public struct HymnLyricsView: View {
+    
+    @ObservedObject private var viewModel: HymnLyricsViewModel
+    
+    init(viewModel: HymnLyricsViewModel) {
+        self.viewModel = viewModel
+    }
+    
+    public var body: some View {
+        guard let lyrics = viewModel.lyrics else {
+            return Text("error!")
+        }
+        
+        guard !lyrics.isEmpty else {
+            return Text("loading...")
+        }
+
+        return Text(lyrics[0].verseContent[0])
```

Comment:
How does this work? I can see that verseContent would need to be an array but why does lyrics have to be an array?

Reply:
A song has a list of verse and each verse is a list of strings (lines). Lyrics = list of verses